### DISCUSSION TITLE
文字方向的判定，碰到单字如何处理？，单字并不是0和180的呢

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#13598] [Q&A]
文字方向的判定，碰到单字如何处理？，单字并不是0和180的呢

**Comment by SWHL** [comment]
文本识别部分支持90和270度文字的

**Reply by cqray1990** [reply]
> 文本识别部分支持90和270度文字的

那文字识别的训练不是要加入单字的四个方向的数据训练?这肯定会影响识别结果吧？
@SWHL

**Reply by SWHL** [reply]
可能会吧，具体需要做实验验证了
